manifests/resource/server.pp

```diff
@@ -592,8 +594,31 @@
   if $ssl {
     # Access and error logs are named differently in ssl template
 
-    File <| title == $ssl_cert or path == $ssl_cert or title == $ssl_key or path == $ssl_key |>
-    -> concat::fragment { "${name_sanitized}-ssl-header":
+    if $ssl_key {
+      if $ssl_key =~ Array {
+        $ssl_key_real = $ssl_key
+      } else {
+        $ssl_key_real = [$ssl_key]
```

Comment:
OK but this is doing the same as what [`flatten()`](https://puppet.com/docs/puppet/7.5/function.html#flatten) does like I suggested, but in a more complicated way. :shrug:

Reply:
Sorry for wasting your time. While trying your suggestion, I checked also for the Boolean type, which isn't required at all.
I've updated the commit again.